verb.el

```diff
@@ -1107,6 +1138,30 @@ This affects only the current buffer."
             (yes-or-no-p "Unset all Verb variables for current buffer? "))
     (setq verb--vars nil)))
 
+(defun verb-load-env-file (filename)
+  "Load a json or elisp configuration file, FILENAME, into verb variables."
+  (interactive)
+  (save-excursion
+    (let* ((file-contents (with-temp-buffer
```

Comment:
Let's just use `load` here and assume that the file is Elisp, no need to check file extension or anything.
I like the unsecure config warning idea 👍🏻

Reply:
See 1 last plea comment about value of JSON.